configure

```diff
@@ -1350,6 +1350,36 @@ def configure_inspector(o):
                        options.without_ssl)
   o['variables']['v8_enable_inspector'] = 0 if disable_inspector else 1
 
+bin_override = None
+
+def setup_bin_overrides():
+  global bin_override
+  # If the system python is not the python we are running (which should be
+  # python 2), then create a directory with a symlink called `python` to our
+  # sys.executable. This directory will be prefixed to the PATH, so that
+  # other tools that shell out to `python` will use the appropriate python
+
+  from distutils.spawn import find_executable as which
+  if os.path.realpath(which('python')) == os.path.realpath(sys.executable):
+    return
+
+  bin_override = os.path.abspath('out/tools/bin')
+  try:
+    os.makedirs(bin_override)
+  except os.error as e:
```

Comment:
For consistency with the rest of the file, can you use `OSError` in this function?